src/pages/workspaces/workspace/Data.js

```diff
@@ -48,22 +49,34 @@ const styles = {
   }
 }
 
-const DataTypeButton = ({ selected, entityName, children, entityCount, iconName = 'listAlt', iconSize = 14, buttonStyle, ...props }) => {
+const DataTypeButton = ({ selected, entityName, children, entityCount, iconName = 'listAlt', iconSize = 14, buttonStyle, after, ...props }) => {
   const isEntity = entityName !== undefined
 
-  return h(Clickable, {
-    style: { ...Style.navList.item(selected), color: colors.accent(1.2), ...buttonStyle },
+  return h(Interactive, {
+    style: { ...Style.navList.itemContainer(selected) },
     hover: Style.navList.itemHover(selected),
-    ...(isEntity ? { tooltip: entityName, tooltipDelay: 250 } : {}),
-    ...props
+    as: 'div',
+    role: 'listitem'
   }, [
-    div({ style: { flex: 'none', display: 'flex', width: '1.5rem' } }, [
-      icon(iconName, { size: iconSize })
-    ]),
-    div({ style: { flex: 1, ...Style.noWrapEllipsis } }, [
-      entityName || children
+    h(Clickable, {
+      style: { flex: '1 1 auto', maxWidth: 232, ...Style.navList.item(selected), color: colors.accent(1.2), ...buttonStyle },
+      ...(isEntity ? {
+        tooltip: entityName ? `${entityName} (${entityCount} row${entityCount === 1 ? '' : 's'})` : undefined,
```

Comment:
Technically, in this instance the tooltip is only necessary in situations where the `noWrapEllipsis` hides some of the content. Otherwise it's basically just duplicative of what the user already sees. 

As far as I have seen, the only cases where this widget displays its children rather than an entity name is for the Reference Data, and the words "Workspace Data" and "Files" at the end. Since these don't extend beyond 280px, a tooltip is technically unnecessary for them.